core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2665,46 +2670,35 @@ write_files:
           namespace: kube-system
           labels:
 
-  - path: /srv/kubernetes/manifests/metrics-server-de.yaml
+  - path: /etc/kubernetes/manifests/metrics-server.yaml
     content: |
-        apiVersion: extensions/v1beta1
-        kind: Deployment
```

Comment:
I was unable to make this work w/ Deployment or DaemonSet.
When this was a deployment/daemonset, `kubectl --namespace kube-system get apiservice v1beta1.metrics.k8s.io` and `kubectl --namespace kube-system delete apiservice v1beta1.metrics.k8s.io` seemed to hang forever.